docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+(insert scope and description here)
+
+Items:
+* [ ] Ran test & live-tests
+* [ ] Implemented (new) unit tests
+* [ ] Removed credentials
+* [ ] Updated [documentation](sdk.up42.com)
```

Comment:
Maybe also add
[ ] Removed argument pointing to staging
?

Reply:
added, ty